samples/aspnetapp/aspnetapp/Scripts/WebForms/Menu.js

```diff
@@ -170,7 +170,6 @@ function Menu_GetData(item) {
         var a = (item.tagName.toLowerCase() == "a" ? item : WebForm_GetElementByTagName(item, "a"));
         var menu = Menu_FindMenu(a);
         try {
-            // CodeQL [SM04509] Code generated from project template
```

Comment:
It looks like these changes from https://github.com/microsoft/dotnet-framework-docker/pull/1136 were removed. These should be added back since the underlying issues aren't fixed.